backend/webdav/api/types.go

```diff
@@ -19,6 +19,9 @@ const (
 	// The same as time.RFC1123 with optional leading zeros on the date
 	// see https://github.com/ncw/rclone/issues/2574
 	noZerosRFC1123 = "Mon, _2 Jan 2006 15:04:05 MST"
+	// The same as time.RFC1123 with suffixed timezone offset
+	// see https://github.com/ncw/rclone/issues/3048
+	TZOffsetRFC1123 = "Wed, 13 Mar 2019 15:19:49 GMT-00:00"
```

Comment:
That doesn't look like a [go magic time value](https://golang.org/pkg/time/#Parse)?  I think you need something like `"Mon, 02 Jan 2006 15:04:05 MST-07:00"`

Reply:
I did wonder how it read a human readable time stamp, makes sense now!
Changing it.